components/raftstore/src/store/fsm/peer.rs

```diff
@@ -1115,6 +1115,7 @@ where
                     "res" => ?res,
                 );
                 self.on_ready_result(&mut res.exec_res, &res.metrics);
+                res.metrics.written_query_num += 1;
```

Comment:
Why `written_query_num += 1` after `on_ready_result`? Should the  `written_query_num` plus 1 before `on_ready_result`?